shared/styles/index.desktop.tsx

```diff
@@ -156,8 +156,8 @@ export const initDesktopStyles = () => {
         `.darkMode .color_${name} {color: ${darkColor};}\n` +
         `.hover_color_${name}:hover {color: ${color};}\n` +
         `.darkMode .hover_color_${name}:hover {color: ${darkColor};}\n` +
-        `.hover_container:hover .hover_contained_color_${name} {color: ${color};}\n` +
-        `.darkMode .hover_container:hover .hover_contained_color_${name} {color: ${darkColor};}\n` +
+        `.hover_container:hover .hover_contained_color_${name} {color: ${color} !important;}\n` +
```

Comment:
@adamjspooner Are you sure this won't affect other styles?

Reply:
IIRC, I looked at all instances where `.hover_container` was being used and didn't see any problems. I can't think of a reason why you'd be using it and not want the hover color to override things, but I'm not certain. I'll take another look, though. It's been a while since I worked on this. Thanks for looking!

Reply:
@thebearjew, yup, everything checks out. I looked at all instances of `.hover_container`.